src/renderer/screens/exchange/Swap2/Form/DrawerTitle.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import React from "react";
+import { Trans } from "react-i18next";
+import Box from "~/renderer/components/Box/Box";
+import { Separator } from "./Separator";
+import Text from "~/renderer/components/Text";
+
+export function DrawerTitle({ i18nKey }: { i18nKey: string }) {
+  return (
+    <>
+      <Box horizontal justifyContent="center">
+        <Text fontSize={6} fontWeight="600" ff="Inter" textTransform="capitalize">
```

Comment:
We usually adhere to the ff attr determining the font weight, that way we can change it everywhere at once if we device SemiBold is no longer 600. In your case this would be ff="Inter|SemiBold" without the need for an explicit